golem-service/config/cloud-server.toml

```diff
@@ -18,6 +18,10 @@ database = "golem"
 [templates.store]
 object_prefix = ""
 
+[worker_connect_cache]
+max_capacity = 1000
```

Comment:
what should be this? Pretty sure people will try to connect to million workers.

Reply:
You cannot open a million tcp connections to a single cloud server :) Leave it as is, we cannot figure out these numbers without the benchmarking framework